### PR TITLE
Fix errors

### PR DESCRIPTION
**Pull request by ckotte**
It's not possible (anymore) to create a box with the current version.

I fixed several errors (not enough memory, missing packages, wrong virtualbox package) and I made some improvements (better script output, removed obsolete config). 

Some commits are mandatory and some are more or less optional, but I didn't want to create several pull requests.

See the commits for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ My goal was to roughly duplicate the attributes from a
 * 512 MB memory
```
Comment (appleby) [inline README.md:19]: should this be updated to 768 MB now?

File: arch-template.json [hunk 2]
```diff
@@ -22,6 +23,8 @@
                 "/usr/bin/curl -O http://{{ .HTTPIP }}:{{ .HTTPPort }}/poweroff.timer<enter><wait5>",
                 "/usr/bin/bash ./enable-ssh.sh<enter>"
             ],
+            "cpus": 1,
+            "memory": 768,
```
Comment (appleby) [inline arch-template.json:27]: \> 512 MB required to boot an install iso?

*\*sad old-timer noises\**

Comment (ckotte) [reply arch-template.json:27]: 512MB is at least the default in packer, but I couldn't boot the current install CD with 512MB. However, it was working in the past. Either it's related to VirtualBox or the ISO.

Comment (appleby) [reply arch-template.json:27]: I believe you. The archlinux install guide claims (oddly specific) 530 MiB RAM is required

> Arch Linux should run on any x86_64-compatible machine with a minimum of 530 MiB RAM.

https://wiki.archlinux.org/index.php/installation_guide#:~:text=Arch%20Linux%20should%20run%20on,working%20internet%20connection%20is%20available.

File: wrapacker [hunk 3]
```diff
@@ -30,256 +30,353 @@ VALID_TIME_UNITS=(ns us ms s m h)
...
-			-only=$PACKER_PROVIDER \\
-			-var "iso_url=$ISO_URL" \\
-			-var "iso_checksum_url=$ISO_CHECKSUM_URL" \\
-			-var "ssh_timeout=$SSH_TIMEOUT" \\
-			-var "country=$country" \\
-			-var "headless=$headless" \\
-			arch-template.json
-	EOF
+if [[ $dry_run = true ]]; then
+  if [[ $force = true ]]; then
```
[the remark this answers is not in the document]
Comment (elasticdog) [reply wrapacker:327]: Great suggestion...I'll put that into place after merging.

**Pull request merged**